index.js

```diff
@@ -498,10 +498,18 @@ prototype.buildOperation = function(oldOperation, operationDefaults) {
     parameters.push(this.buildParameter(oldParameter));
   });
 
+  var tags = (operationDefaults.tags || []).concat(oldOperation.tags || []);
+
+  tags = tags.filter(function(e, i, arr) {
```

Comment:
Ok, I can user "accepted" instead of "element". It's ok?

The test for style guides fails if I do:

```javascript
 return arr.lastIndexOf(e) === i;
```